src/printf.py

```diff
@@ -5,6 +5,8 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("%i %.2f %s" %(x, y, z))
 
+# Use the 'format' string method to print the same thing
 
-# Use the 'format' string method to print the same thing
\ No newline at end of file
+print("{} {} {}").format(x, y, z)
```

Comment:
Does this work for you?

Reply:
yea its the new way of doing it

Reply:
Did you run it? I'm just confused why it didn't yell at you.